src/Identity/Core/src/PasskeyOptions.cs

```diff
@@ -0,0 +1,124 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.AspNetCore.Identity;
+
+/// <summary>
+/// Specifies options for passkey requirements.
+/// </summary>
+public class PasskeyOptions
```

Comment:
I wonder if we should consider renaming `DefaultPasskeyHandler` to `PasskeyHandler` and `PasskeyOptions` to `PasskeyHandlerOptions`. That might make the correlation between the implementation and the options clearer. This is similar to `SecurityStampValidator` and `SecurityStampValidatorOptions` which exist today.

Reply:
I like removing "Default" from the handler name. This aligns with most other services like the SecurityStampValidator. I'm not sure about renaming PasskeyOptions though. To me, it's analogous to PasswordOptions. I know there's no PasswordHandler that has to read it, so this is a little different, but we've also removed "Handler" from other options types that could have it. For example, most of our auth handlers are that way (e.g. CookieAuthenticationOptions/CookieAuthenticationHandler, JwtBearerOptions/JwtBearerHandler, etc.)